cli/mender-artifact/write.go

```diff
@@ -182,6 +183,16 @@ func writeRootfs(c *cli.Context) error {
 		}
 	}
 
+	go func() {
+		str := fmt.Sprintf("Writing: %s", <-aw.State)
+		for {
+			fmt.Printf(str)
+			str = fmt.Sprintf("Writing: %s", <-aw.State)
+			fmt.Printf(" \u2713\n")
+		}
```

Comment:
This is a memory leak (no exit condition).

Reply:
Hmm, yes. But the application exits right after, is it really required?
Nvm. I know the answer to this :p